packages/agoric-cli/src/helpers.js

```diff
@@ -3,6 +3,8 @@
 
 /** @import { ChildProcess } from 'child_process' */
 
+export { getNetworkConfig } from './lib/network-config.js';
+
```

Comment:
Not used in this PR.  Fixup to base?

Reply:
It's imported in a3p-integration/proposals/z:acceptance/test-lib/psm-lib.js.

Reply:
oops misread.